MatrixSDK/Data/MXRoomState.m

```diff
@@ -334,15 +334,30 @@ - (NSString *)creatorUserId
     
     // Check it from the state events
     MXEvent *event = [stateEvents objectForKey:kMXEventTypeStringRoomCreate].lastObject;
+    NSString* sender = [event sender];
     
+    if (event && sender)
+    {
+        creatorUserId = [sender copy];
```

Comment:
This used to fetch the creatorID from 
```
MXJSONModelSetString(creatorUserId, eventContent[@"creator"]);
creatorUserId = [creatorUserId copy];
```

are you sure that's not needed anymore?